### DISCUSSION TITLE
Logging from a Tornado handler

### DISCUSSION THREAD

**Discussion by martynia** [DIRACGrid/DIRAC#6361] [Q&A] [answered]
Hi,

My logging level in my Tornado handler seems to be `info`. Debug records don't show up. I have enabled `LogLevel = DEBUG` both in my handler section (`/Systems/WorkloadManagement/Devel/Services/TornadoPilotLogging/LogLevel`  in my case) and in desperation ;-) in the global Tornado section as well:` /Systems/Tornado/Devel/LogLevel`.

The code follows a template, basically:

```python
sLog = gLogger.getSubLogger(__name__)


class TornadoPilotLoggingHandler(TornadoService):
    log = sLog

    @classmethod
    def initializeHandler(cls, infoDict):
        """
        Called once, at the first request. Create a directory where pilot logs will be stored.

        :param infoDict:
        :return: None
        """

        cls.log.info("Handler initialised ...")
        cls.log.debug("with a dict: ", str(infoDict))
```

The debug statement never comes up. Is the `LogLevel` configuration option taken into account at all ?

 Thanks, JM

**Comment by chaen** [comment]
what version are you running ?

**Reply by chaen** [reply]
You are right, there is a generic issue. I'll open a github issue and follow up

**Reply by martynia** [reply]
The version is v7r2p15.

**Reply by marianne013** [reply]
Why are you using v7r2 ???????

**Reply by martynia** [reply]
Well,

I blindly copied what System Administration tab lists.

**Reply by marianne013** [reply]
I recommend git status ("HEAD detached at v7.3.20"). When you run this is development mode, the version displayed doesn't change, and I can't remember how to update it, which is probably a good thing, otherwise you'd be now running the "morningjanusz' version.

**Comment by fstagni** [comment, marked as answer]
Being fixed in https://github.com/DIRACGrid/DIRAC/pull/6373

**Comment by martynia** [comment]
Well,

I blindly copied what System Administration tab lists.

JM
